src/logger.js

```diff
@@ -0,0 +1,34 @@
+import fs from 'fs';
+
+let LOG_FILE = './output/log.txt';
```

Comment:
Why add something custom instead of using a tested maintained logging framework?